staging/src/k8s.io/pod-security-admission/cmd/webhook/server/server.go

```diff
@@ -54,9 +55,14 @@ func NewServerCommand() *cobra.Command {
 		Long: `The PodSecurity webhook is a standalone webhook server implementing the Pod
 Security Standards.`,
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			return runServer(cmd.Context(), opts)
+			if err := runServer(cmd.Context(), opts); err != nil {
+				fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
I just moved the func `Fprintf` from `main` to `RunE`, for I found if I put the`Fprintf` in `main`, when some errors happen in `command.Execute()`, information of usage would be printed firstly then error messages. see https://github.com/kubernetes/kubernetes/blob/b35eec2332b2f4abb133ede058956ac79f6842c8/vendor/github.com/spf13/cobra/command.go#L974-L994. `RunE` function is called by  `err = cmd.execute(flags)`, if some errors happen in `RunE`,  they should be printed first, `c.Println(cmd.UsageString())` would be called then. correct me if not.